src/explicitode.jl

```diff
@@ -193,6 +225,9 @@ function taylorstep!(f, t::Taylor1{T}, x::Taylor1{U}, t0::T, t1::T, order::Int,
 
     # Compute the step-size of the integration using `abstol`
     δt = stepsize(x, abstol)
+    if isinf(δt)
+        δt = _second_stepsize(x, abstol)
```

Comment:
Just for consistency: is there any particular reason why `_second_stepsize` is called from `stepsize` in the vector case, but called from `taylorstep!` in the scalar case?

Reply:
Fair point; I guess is speed. In the vector case, `_second_stepsize` is called if the `min` of all the possible stepsizes is `Inf` (so only once), which I guess occurs seldom. In the scalar case, you have to check it consistently. Somewhere here was actually the problem that lead to the performance regression.